docs/design/ha-flow/ha-flow-design.md

```diff
@@ -0,0 +1,76 @@
+# HA-flows
```

Comment:
HA is high availability? I suggest to add the description for the abbreviature in the beginning.

Reply:
fixed